kernel/kernel-api/src/main/java/io/delta/kernel/internal/SnapshotImpl.java

```diff
@@ -155,6 +152,7 @@ public SnapshotReport getSnapshotReport() {
   /**
    * Get the domain metadata map from the log replay, which lazily loads and replays a history of
    * domain metadata actions, resolving them to produce the current state of the domain metadata.
+   * Only active domain metadata should be in this map.
```

Comment:
nit - "are included" or something instead of "should"; the docs should be a promise not an instruction